src/operator/src/req_convert/insert/stmt_to_region.rs

```diff
@@ -65,6 +65,11 @@ impl<'a> StatementToRegion<'a> {
         let table_schema = table.schema();
         let table_info = table.table_info();
 
+        ensure!(
+            !catalog::consts::is_readonly_schema(schema.as_str()),
```

Comment:
Prefer to use `&schema` instead of `schema.as_str()` in convention, other places as well.